readme.md

```diff
@@ -1,5 +1,19 @@
+⚠️ This Repository is Deprecated
+---
+
+Sashimi are deprecated in favour of a new architecture for developing steps that focuses on providing a simplified development experience in a language other than .NET, named [Step Packages](https://github.com/OctopusDeploy/step-api/blob/main/docs/StepPackages.md).
+
+In 2022, we found that the costs of having Sashimi flavours separated out across multiple repositories were outweighing the intended benefits, so we invested in reducing these development costs by consolidating Sashimi repositories. 
+As the repository that contains the shared infrastructure for Sashimi flavours, this repository was consolidated into [OctopusDeploy](https://github.com/OctopusDeploy/OctopusDeploy) and [Calamari](https://github.com/OctopusDeploy/Calamari).
+This repository is now only for bug-fixing.
```

Comment:
Minor nit, aiming to give a little more detail on the context of where changes to this repository are applicable might help.

> This repository is now only accepting bug fixes for the 2022.2 and 2022.1 LTS branches until their EOL.